src/ProgressBar.js

```diff
@@ -89,22 +83,24 @@ class ProgressBar extends React.Component {
     label,
     srOnly,
     striped,
-    active,
+    animated,
     className,
     style,
+    bsStyle,
+    bsPrefix,
     ...props
   }) {
-    const [bsProps, elementProps] = splitBsProps(props);
 
     const classes = {
-      ...getClassSet(bsProps),
-      active,
-      [prefix(bsProps, 'striped')]: active || striped
+      [bsPrefix]: true,
+      [`bg-${bsStyle}`]: bsStyle,
```

Comment:
let's change `bsStyle` to `variant` to be consistent with the new styling names :)